app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="post-container">
+  <span class="post-title"><%= post.title %></span>
+  <span class="post-bio"><%= post.text %></span>
+  <span class="post-totalpost">Comments : <%= post.comments_counter %>, Likes : <%= post.likes_counter %></span>
+</div>
```

Comment:
- [ ] Please for good user experience; kindly make the individual post to be a clickable element that can lead us to its `show_page`. I was only able to access the `post_show` page via the url. 
- [ ] Kindly ensure that it is designed in accordance to the wirefreme ( the number of likes is not properly placed).

<img width="300" alt="Untitled" src="https://github.com/rbd3/Blog-App2/assets/103386183/d7dfcdbe-f128-4d52-8a1b-834b878a0c45">